### DISCUSSION TITLE
v0.9.6-beta.0

### DISCUSSION THREAD

**Discussion by thanksmister** [thecowan/wallpanel-android#313] [Announcements]
This version uses a new MQTT library that is compatible with Android 9 and newer devices.   This version requires a minimum Android SDK version of 21 (previously 19 min).   

## Installation

Install the APK by side-loading onto your device using the Android ADB command line tool or download the APK file directly to your device and open it. 

<hr /><em>This discussion was created from the release <a href='https://github.com/thanksmister/wallpanel-android/releases/tag/v0.9.6.0'>v0.9.6-beta.0</a>.</em>

**Comment by TheQue42** [comment]
Am I to understand this as the next full release of the application will no longer support any device with an android version less that 9? Thats quite a jump from "Android Device running Android OS 4.4 or greater"?

**Comment by thanksmister** [comment]
This supports back to Android SDK 21 but offers MQTT support for those running Oreo.  Currently the app supports back to Android SDK 19.  So if you need to support older devices you can install previous release versions.  I will clearly mark which version supports which Android SDK.

**Comment by thanksmister** [comment]
Android version 9 is not the same as SDK version 21.  It’s confusing I know.

**Comment by thanksmister** [comment]
Here is info that references version to SDK https://developer.android.com/studio/releases/platforms
